Makefile

```diff
@@ -1,10 +1,29 @@
 .PHONY: test
+all: test-catalog test-jmrl test-article
 
-test:
-	rspec tests/test-*.rb
+test-catalog:
```

Comment:
The "test" target used to run all the tests.  Instead you have an all target now (which is fine) but it doesn't really test all the stuff.

Since we can no longer test everything we want to test with the test/-*.rb pattern of files, we should drop that target and explicitly have the following targets:
all: test-pools test-master test-client
test-pools: test-pool-catalog test-pool-jmrl test-pool-images test-pool-articles
test-pool-catalog:
test-pool-jmrl:
test-pool-images:
test-pool-articles